### ISSUE TITLE
[a11y] Screen reader does not read description on Send Tab onboarding

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#2915]
## Steps to reproduce
- Clean install, open app

### Expected behavior
Screen reader reads entire screen including description text, "Send a tab from Firefox..."

### Actual behavior
Screen reader only reads title & button text.

### Device information
* Fire TV device: ?
* Latest affected Firefox version: v4.5+ master 2cc87063e1808eaaa291b818cb22c952c9469607
* Earliest affected Firefox version: ?
* Turbo Mode: ?
